docs/Antora/modules/howto_guides/pages/howto_configure_instance.adoc

```diff
@@ -0,0 +1,291 @@
+====
+
+== Configure openDAQ(TM) Instance Module Manager
+By default, the Instance Module Manager uses the current path for loading modules. In the Instance Builder, developers can set a custom module path or replace the default Module Manager with a custom implementation inherited from IModuleManager.
+[tabs]
+====
+Cpp::
++
+[source,cpp]
+----
+#include <opendaq/opendaq.h>
+#include <iostream>
+
+
+int main(int argc, const char* argv[])
+{
+    daq::InstanceBuilderPtr instanceBuilder = daq::InstanceBuilder().setModulePath("/path/to/modules");
+    daq::InstancePtr instance = instanceBuilder.build();
+
+    return 0;
+}
+----
+Overriding module path
+====
+
+[tabs]
+====
+Cpp::
++
+[source,cpp]
+----
+#include <opendaq/opendaq.h>
+#include <iostream>
+
+class CustomMuduleManagerImpl : public ImplementationOfWeak<IModuleManager>
+{
+    CustomMuduleManagerImpl() = default;
+    // implementation
+} 
+
+int main(int argc, const char* argv[])
+{
+    daq::ModuleManagerPtr moduleManager = daq::ModuleManagerPtr(CustomMuduleManagerImpl());
+    daq::InstanceBuilderPtr instanceBuilder = daq::InstanceBuilder().setModuleManager(moduleManager);
+    daq::InstancePtr instance = instanceBuilder.build();
+
+    return 0;
+}
+----
+Setting custom module manager
+====
+
+== Configure openDAQ(TM) Instance Scheduler
+By default, the Instance creates a Scheduler with a number of workers equal to the maximum physical threads. To manually change this amount, developers can use the Instance Builder method `setSchedulerWorkerNum`.
+[tabs]
+====
+Cpp::
++
+[source,cpp]
+----
+#include <opendaq/opendaq.h>
+#include <iostream>
+
+int main(int argc, const char* argv[])
+{
+    daq::InstanceBuilderPtr instanceBuilder = daq::InstanceBuilder().setSchedulerWorkerNum(2);
+    daq::InstancePtr instance = instanceBuilder.build();
+
+    return 0;
+}
+----
+====
+As in previous examples, developers can implement their own solution for the scheduler and use it in the Instance Builder.
+[tabs]
+====
+Cpp::
++
+[source,cpp]
+----
+#include <opendaq/opendaq.h>
+#include <iostream>
+
+class CustomSchedulerImpl : public ImplementationOfWeak<IScheduler>
+{
+    CustomSchedulerImpl(size_t workers)
+    {
+        // initializing
+    }
+    // implementation
+} 
+
+int main(int argc, const char* argv[])
+{
+    daq::SchedulerPtr scheduler = daq::SchedulerPtr(CustomSchedulerImpl(2));
+    daq::InstanceBuilderPtr instanceBuilder = daq::InstanceBuilder().setScheduler(scheduler);
+    daq::InstancePtr instance = instanceBuilder.build();
+
+    return 0;
+}
+----
+====
+
+== Configure openDAQ(TM) Root Device
+If, after creating the Instance object, the developer has not set the root device, the Instance will use the Client device as the root. Using the Instance Builder, developers can discover and connect to available devices using the method `getAvailableDevices`. In the Instance Builder, developers can forcibly set the available device.
```

Comment:
These two sentences feel a bit off. The Client device is the default root that queries the module manager when adding/searching for function blocks/devices. The builder allows a user to specify a different root device that replaces the default, or to change the local ID and info of the default.